packages/remediations/src/newSteps/review.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState, Fragment, useEffect } from 'react';
+import propTypes from 'prop-types';
+import useFormApi from '@data-driven-forms/react-form-renderer/dist/esm/use-form-api';
+import {
+    Text,
+    TextContent,
+    Stack,
+    StackItem
+} from '@patternfly/react-core';
+import { Table, TableVariant, TableHeader, TableBody, sortable } from '@patternfly/react-table';
+import { CloseIcon, ExclamationTriangleIcon, RedoIcon } from '@patternfly/react-icons';
+import { Button } from '@patternfly/react-core/dist/esm/components/Button/Button';
```

Comment:
No need to import PF components directly from esm. Just use the shorthand as with the other react core imports.